test/functional/interface_bitcoin_cli.py

```diff
@@ -52,7 +51,7 @@ def run_test(self):
         self.log.info("Test -getinfo returns expected network and blockchain info")
         if self.is_wallet_compiled():
             self.nodes[0].encryptwallet(password)
-        cli_get_info = self.nodes[0].cli('-getinfo').send_cli()
+        cli_get_info = self.nodes[0].cli().send_cli('-getinfo')
```

Comment:
same here

Reply:
Changed to improve the debug log output:
```diff
- TestFramework.bitcoincli (DEBUG): Running bitcoin-cli command: None
+ TestFramework.bitcoincli (DEBUG): Running bitcoin-cli command: -getinfo

- TestFramework.bitcoincli (DEBUG): Running bitcoin-cli command: None
+ TestFramework.bitcoincli (DEBUG): Running bitcoin-cli command: -version
```
Can revert in #18594 that also touches the testfile.

Reply:
I changed it to improve the debug log output:
```diff
- TestFramework.bitcoincli (DEBUG): Running bitcoin-cli command: None
+ TestFramework.bitcoincli (DEBUG): Running bitcoin-cli command: -getinfo

- TestFramework.bitcoincli (DEBUG): Running bitcoin-cli command: None
+ TestFramework.bitcoincli (DEBUG): Running bitcoin-cli command: -version
```
I can revert it in #18594 that also touches the testfile or look at improving the logging output for these cases.

Reply:
That should be fixed by logging the options, not passing the command line option as the rpc command

Reply:
> That should be fixed by logging the options, not passing the command line option as the rpc command

Done in #18712.